### DISCUSSION TITLE
SSO OIDC Error: {"detail":"The email or password provided is incorrect. Please check for typos and try logging in again."}

### DISCUSSION THREAD

**Discussion by badroshdy** [open-webui/open-webui#6223] [General]
Hi,
I am trying to configure Open WebUI SSO with OIDC. After the configuration, I see the SSO button, once I click it, I get transfered to our ADFS server and then once the redeirect happens in Open WebUI, I am getting this error:  Error: {"detail":"The email or password provided is incorrect. Please check for typos and try logging in again."}.

I think I provided all needed  Provided info in open webui: enable_oauth_signup = true, oauth_merge_accounts_by_email = true, oauth_client_id, oauth_client_secret, openid_provider_url, openid_redirect_uri, and the rest left as default. 

I checked splunk and ADFS logs but no issues. 

Am I missing authorization in the application? Any idea about this isse?

**Comment by ArnOCP** [comment]
Hi badroshdy,
I saw you can redirect to the ADFS site when you click on the SSO button.
How do you do this, i have an erro 500 when i click on the button on my site.
Regards,

**Comment by Itsanewday** [comment]
> Hi, I am trying to configure Open WebUI SSO with OIDC. After the configuration, I see the SSO button, once I click it, I get transfered to our ADFS server and then once the redeirect happens in Open WebUI, I am getting this error: Error: {"detail":"The email or password provided is incorrect. Please check for typos and try logging in again."}.
> 
> I think I provided all needed Provided info in open webui: enable_oauth_signup = true, oauth_merge_accounts_by_email = true, oauth_client_id, oauth_client_secret, openid_provider_url, openid_redirect_uri, and the rest left as default.
> 
> I checked splunk and ADFS logs but no issues.
> 
> Am I missing authorization in the application? Any idea about this isse?

Hi, I don't understand SSO well. Could you explain how you configured it to enable SSO support in OpenWebUI?
